internal/plugins/info.csv

```diff
@@ -0,0 +1,250 @@
+redpanda_data_transform ,processor ,redpanda_data_transform ,certified  ,n          ,n     ,n
+reject                  ,output    ,reject                  ,certified  ,n          ,y     ,y
+reject_errored          ,output    ,reject_errored          ,certified  ,n          ,y     ,y
+resource                ,input     ,resource                ,certified  ,n          ,y     ,y
+resource                ,output    ,resource                ,certified  ,n          ,y     ,y
+resource                ,processor ,resource                ,certified  ,n          ,y     ,y
+retry                   ,output    ,retry                   ,certified  ,n          ,y     ,y
+retry                   ,processor ,retry                   ,certified  ,n          ,y     ,y
+ristretto               ,cache     ,ristretto               ,community  ,n          ,y     ,y
+schema_registry         ,input     ,schema_registry         ,enterprise ,n          ,y     ,y
+schema_registry         ,output    ,schema_registry         ,enterprise ,n          ,y     ,y
+schema_registry_decode  ,processor ,schema_registry_decode  ,certified  ,n          ,y     ,y
+schema_registry_encode  ,processor ,schema_registry_encode  ,certified  ,n          ,y     ,y
+select_parts            ,processor ,select_parts            ,certified  ,n          ,y     ,y
+sentry_capture          ,processor ,sentry_capture          ,community  ,n          ,n     ,n
+sequence                ,input     ,sequence                ,certified  ,n          ,y     ,y
+sftp                    ,input     ,sftp                    ,certified  ,n          ,y     ,y
+sftp                    ,output    ,sftp                    ,certified  ,n          ,y     ,y
+skip_bom                ,scanner   ,skip_bom                ,certified  ,n          ,y     ,y
+sleep                   ,processor ,sleep                   ,certified  ,n          ,y     ,y
+snowflake_put           ,output    ,snowflake_put           ,enterprise ,n          ,y     ,y
+socket                  ,input     ,socket                  ,certified  ,n          ,n     ,n
+socket                  ,output    ,socket                  ,certified  ,n          ,n     ,n
+socket_server           ,input     ,socket_server           ,certified  ,n          ,n     ,n
+split                   ,processor ,split                   ,certified  ,n          ,y     ,y
+splunk                  ,input     ,splunk                  ,enterprise ,n          ,y     ,y
+splunk_hec              ,output    ,splunk_hec              ,enterprise ,n          ,y     ,y
+sql                     ,cache     ,sql                     ,certified  ,n          ,n     ,n
+sql                     ,output    ,sql                     ,community  ,y          ,n     ,n
+sql                     ,processor ,sql                     ,community  ,y          ,n     ,n
+sql_insert              ,output    ,sql_insert              ,certified  ,n          ,y     ,y
```

Comment:
We need some more data in this table.

`sql_*` connectors have different support levels for different drivers. For example the ClickHouse driver is community whereas the PostgreSQL driver is certified.

Similarly the `cache` outputs have different support levels depending on the target cache.

Reply:
The individual cache output support levels are covered by the `cache` type, those are the underlying components being used.

For `sql` drivers I think maybe the way forward would be to add a specific `sql_driver` type as it's redundant to have each driver represented for each sql type. Also, having a given plugin/type combo like `sql_insert` represented multiple times in the table will be effectively seen as duplicates on the back end.

Reply:
> The individual cache output support levels are covered by the cache type, those are the underlying components being used.

Thanks!

> For sql drivers I think maybe the way forward would be to add a specific sql_driver type as it's redundant to have each driver represented for each sql type. Also, having a given plugin/type combo like sql_insert represented multiple times in the table will be effectively seen as duplicates on the back end.

Will this be handled as part of this PR?

Reply:
@JakeSCahill yep, done. Going to merge as is but if you take a look and want changes just let me know and I'll follow up.